ray cluster instructions added (#32)
Bodo-inc/Bodo-examples: examples/tpch/README.md

`python dask_queries.py --folder s3://[bucket-name]/SF100 --scheduler-file [Path]/scheduler.json --worker 288`

## Modin on Ray

### Installation

Start with installing Ray in a new conda environment.

```

> - conda create --name ray

Reviewer: Could you remove `-` in front  of all the commands ?

- conda activate ray
- pip install -U "ray[default]"
- pip install "modin[all]"
- pip install s3fs
- pip install -U boto3
```

### Running queries on single node

Update the path with your data path in the `main()` function and add your AWS credentials to `ray_queries.py` script.

Run the script with `python ray_queries.py` on single node.

### Running queries on multi nodes

> A multi nodes cluster needs to be configured  with `modin.yaml` file which is available in this folder as an example. Please add your AWS credentials under `setup_commands` and update the `InstanceType` under the `node config` and `initial_workers` sections with the cluster type and number of the nodes. For this benchmark we used 16 nodes `c5n.18xlarge` cluster. 

Reviewer: ```suggestion
A multi nodes cluster needs to be configured  with `modin.yaml` file which is available in this folder as an example. Please add your AWS credentials under `setup_commands` and update the `InstanceType` under the `node config` and `initial_workers` sections with the cluster type and number of the nodes. 
```
We don't need to mention 16 nodes information here.

> A multi nodes cluster needs to be configured  with `modin.yaml` file which is available in this folder as an example. Please add your AWS credentials under `setup_commands` and update the `InstanceType` under the `node config` and `initial_workers` sections with the cluster type and number of the nodes. For this benchmark we used 16 nodes `c5n.18xlarge` cluster. 

Reviewer 2: ```suggestion
A multi-node cluster needs to be configured with a yaml script which is available in this folder as an example (`modin.yaml`). Please add your AWS credentials under `setup_commands` and update the `InstanceType` under the `node config` and `initial_workers` sections with the cluster type and number of the nodes. For this benchmark we used 16 nodes `c5n.18xlarge` cluster. 
```

> A multi nodes cluster needs to be configured  with `modin.yaml` file which is available in this folder as an example. Please add your AWS credentials under `setup_commands` and update the `InstanceType` under the `node config` and `initial_workers` sections with the cluster type and number of the nodes. For this benchmark we used 16 nodes `c5n.18xlarge` cluster. 

Reviewer 2: ```suggestion
A multi-node cluster needs to be configured with a yaml script which is available in this folder as an example (`modin.yaml`). Please add your AWS credentials under `setup_commands` and update the `InstanceType` under the `node config` and `initial_workers` sections with the cluster type and number of the nodes. For this benchmark we used 16 nodes `c5n.18xlarge` cluster. 
```

Reviewer 2: I would also put a link to their own documentation or tutorial since it will be more up-to-date and complete.

Reviewer 2: Please update this paragraph when you switch to the new yaml file that I sent you.

> To start your cluster run the below command:

Reviewer: Add a newline before this one.

```
- ray up modin.yaml
```

> The creation of nodes may take long, track the status of your instances from your EC2 console. You may bring down the cluster any time by terminating directly from your EC2 console or running the below command :

Reviewer: ```suggestion
```

### Running queries on multi nodes

A multi nodes cluster needs to be configured  with `modin.yaml` file which is available in this folder as an example. Please add your AWS credentials under `setup_commands` and update the `InstanceType` under the `node config` and `initial_workers` sections with the cluster type and number of the nodes. For this benchmark we used 16 nodes `c5n.18xlarge` cluster. 
To start your cluster run the below command:
```
- ray up modin.yaml
```

> The creation of nodes may take long, track the status of your instances from your EC2 console. You may bring down the cluster any time by terminating directly from your EC2 console or running the below command :

Reviewer: We don't need to say the first sentence `The creation of nodes may take long, track the status of your instances from your EC2 console.`.
Move `bring down the cluster any time by terminating directly from your EC2 console or running the below command :`  and the command to after the running step.

```
- ray down modin.yaml
```

> Now, run `ray_queries_ray_cluster.py` script with the below command:

Reviewer: ```suggestion
Once cluster is up, run `ray_queries_ray_cluster.py` script with the following command:
```
